7_deploy_osh/play.yml

```diff
@@ -1,4 +1,17 @@
 ---
+
+- hosts: osh-deployer
+  gather_facts: yes
+  any_errors_fatal: true
+  roles:
+    - suse-dev-patcher
+
+- hosts: osh-deployer
+  gather_facts: yes
+  any_errors_fatal: true
+  roles:
+    - { role: suse-build-images, when: build_osh_images | default('no') == 'yes' }
```

Comment:
I'd totally prefer if that was given in non one-liner format. Also, moving towards (build_osh_images | default(False)) | bool is less error prone, should build_osh_images be provided.

Reply:
What format you are looking for non one-liner format.

Is it something like this? If not, please clarify.
- { role: suse-build-images, 
    when: build_osh_images | default (False) | boolean }

Reply:
you can remove the `{` `,` and `}`.